roles/commcare_analytics/templates/nginx/superset_nginx_without_ssl_config.j2

```diff
@@ -23,6 +25,8 @@ server {
 
   client_max_body_size 4G;
 
+  limit_req zone=ip_limited burst=100 nodelay;
```

Comment:
note:

We are assigning the zone to this server configuration. Allowing a burst of 100 to handle when excessive requests are sent and so we can maintain a queue of 100.
The "nodelay" flag is to ask nginx to forward the burst requests to be processed but still holding their spot in the queue for the necessary time in order to avoid delay while still accounting for the excess load of requests overall.

Reply:
It took a while for me to unpack this. The docs are here: https://nginx.org/en/docs/http/ngx_http_limit_req_module.html

`nodelay` means do not delay excessive requests while requests are being limited.

I think that means that they will be processed. Is that right? Or will they immediately get a 429 response? Or will they immediately be dropped and get no response? ... Or something else?

Regardless, I'm happy with these changes.

Reply:
The excess limit that can fit in the burst limit will be sent for processing right away with their slot still held for time management.
Any in excess would get back a 503.

Better explained in the documentation as

> With the nodelay parameter, NGINX still allocates slots in the queue according to the burst parameter and imposes the configured rate limit, but not by spacing out the forwarding of queued requests. Instead, when a request arrives “too soon”, NGINX forwards it immediately as long as there is a slot available for it in the queue. It marks that slot as “taken” and does not free it for use by another request until the appropriate time has passed

Reply:
> It took a while for me to unpack this

Its definitely not straight forward but is a powerful set of configurations.